tms_objects2.erb

```diff
@@ -586,11 +586,30 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:conceptID lido:source="ATT" lido:type="local"><%= ur["Expr1"] %></lido:conceptID>
               <lido:term><%= ur["TextType"] %></lido:term>
             </lido:eventType>
-          <% if ur["TextType"] == "Gallery label" or ur["AuthorConID"] == '' %>
+          <% if ur["TextType"] == "Gallery label" or ur["AuthorConID"].nil? %>
           <% else %>
+            <lido:eventActor>
+              <lido:displayActorInRole/>
+              <lido:actorInRole>
+                <lido:actor lido:type="person">
+                <lido:actorID lido:source="YCBA" lido:type="local">ycba_actor_<%= ur["AuthorConID"] %></lido:actorID>
+                <lido:nameActorSet>
+                  <lido:appellationValue lido:pref="preferred"><%= ur["AuthorName"] %></lido:appellationValue>
+                </lido:nameActorSet>
+                </lido:actor>
+                <lido:roleActor>
+                  <lido:conceptID></lido:conceptID>
```

Comment:
this roleActor.conceptID needs to be implemented in the template like:
https://github.com/ycba-cia/coboat-github/blob/main/bac-prd-db1/config/LIDOsecond_pass.tpl#L814
so it displays something like:
`<lido:conceptID lido:source="AAT" lido:type="IRI" lido:label="Published catalog entry role">300025492</lido:conceptID>`

Reply:
Just spotted one earlier typo, might as well fix here,  "ATT" should be "AAT"

<lido:conceptID lido:source="ATT" lido:type="local"><%= ur["Expr1"] %></lido:conceptID>